### PR TITLE
Fabo/fix e2e

### PR DESCRIPTION
**Pull request by faboweb**
Working
Removed some smelly code and fixed internationalization

### CODE REVIEW

**Comment by Bitcoinera** [comment]
It seems they are already fixed anyway? 🤷‍♀ 😅

File: tests/e2e/local-chrome.conf.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ const nightwatch_config = {
         "disable-web-security",
         "ignore-certificate-errors",
         "no-sandbox",
-        "headless",
+        // "headless", // headless is not possible as it doesn't allow to fix the internationalization
```
Comment (Bitcoinera) [inline tests/e2e/local-chrome.conf.js:31]: O.o Could you please explain this bit?

Comment (faboweb) [reply tests/e2e/local-chrome.conf.js:31]: a number like `0.4` is displayed in Germany `0,4` so when you compare both in the UI it would fail.

**Comment by jbibla** [comment]
> Removed some smelly code 

👃

**Comment by jbibla** [comment]
> It seems they are already fixed anyway? 

still some improvements in here!

File: tests/e2e/local-chrome.conf.js (hunk 1 above)
Comment (Bitcoinera) [reply tests/e2e/local-chrome.conf.js:31]: oh so interesting!

**Pull request merged**